index.html

```diff
@@ -25,10 +25,12 @@
         </div>
 
         <!--* nav links -->
-        <ul>
-          <li></li>
-          <li></li>
-          <li></li>
+        <ul
+          class="hidden md:flex md:w-[20%] md:justify-between md:items-center md:text-[15px]"
+        >
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
```

Comment:
- [ ] I think it would be better to replicate Figma design by using the same background on your navbar Remember it is important that you try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=lm66x04u5euYTBoh-0) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

**Your design**
![image](https://user-images.githubusercontent.com/102757126/206836606-c5812df5-3347-4928-bba4-2ff52a95872b.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/206836675-357e9fbf-5c92-409f-8c61-e4778e24bf37.png)